File: testcases/paper/Constructors/Constructors.java
public class Constructors {
    int i, j;

    public Constructors() {
        i = 1;
    }

    public void method() {
        j = 2;
    }

    public static void main(String[] args) {
        return;
    }
}

// Constructors don't count: LCOM4 = 1 (+1 if static counts)
// Constructors count:       LCOM4 = 2 (+1 if static counts)

File: testcases/paper/Constructors/Constructors.cpp
class Constructors {
    int i, j;
    
    Constructors() {
        i = 1;
    }

    Constructors(int _i) : i(_i) {}

    Constructors(double _j) : j(_j) {}

    void method() {
        j = 2;
    }
};

// Constructors don't count: LCOM4 = 1
// Constructors count:       LCOM4 = 2